### PR TITLE
feat(storefront): Style Menypage Layout, Sidebar and Header

### LINKED ISSUE #1670: Storefront: Design adjustments for the MenuPage layout
I also did some changes to the SidebarMenu and Header to test and make the design is correct "as a package". Also removed the border in the top-bar. The front page looks fine without it.

Top-bar:
* Endra understrek i topp-menyen til svart og la til 500 font-vekting på teksten slik at den matcher understreken bedre.
* Fjerna understrek i top-baren. Gå i ein litt meir moderne designretning der innholdet aligner, og ikkje masse svarte strekar.
* Laga mindre spacing mellom top-bar og innholdet.

Venstremeny:
* Gjorde venstremenyen litt luftigare og symmetrisk.
* La til lys-grå bakrunnsfarge på aktiv element i sidemenyen for å lage bedre kobling til innholdet.
* Endra til svart aktiv farge på strek i sidemenyen.

Menupage malen:
* Gjorde den grå header-boksen på sida meir symmetrisk med padding.
* La til ikon og diamant på høgre sida.
* Endra breidda på bodyteksten frå 800px til 740px.
* Gjekk ned eit hakk i heading-størrelse på headingane for å skape ein meir balansert typografi på sida.

### PR DESCRIPTION
**Pull request by Thuneer**
Resolves #1670

### CODE REVIEW

**Review by Barsnes — CHANGES_REQUESTED** [review]
Since this PR covers the same area, can we make the breakpoints for these the same? 
image
image

Currently the menu goes above the content before the "Vis side meny" button shows up.

Other than that, everything seems to be in order 🚀

**Comment by Thuneer** [comment]
@Barsnes Good catch on the sidebar menu! I made sure that the NavPageMenu (/grunnleggende) and the PageMenu layouts both work correctly with the sidebar collapsing.

File: apps/storefront/pages/grunnleggende/designelementer/skygger.mdx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { MenuPageLayout } from '@layouts';
 import { Meta, TokenList } from '@components';
+import { MoonIcon } from '@navikt/aksel-icons';
```
Comment (mimarz) [inline apps/storefront/pages/grunnleggende/designelementer/skygger.mdx:3]: Some linting errors here

File: apps/storefront/pages/grunnleggende/designelementer/storrelser-og-avstander.mdx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { MenuPageLayout } from '@layouts';
 import { Meta, TokenList, CodeSnippet } from '@components';
+import { RulerIcon } from '@navikt/aksel-icons';
```
Comment (mimarz) [inline apps/storefront/pages/grunnleggende/designelementer/storrelser-og-avstander.mdx:3]: linting warning also here

**Review by mimarz — APPROVED** [review]
Some linting warnings (didnt mark all) but apart from that looks ok 👍

**Review by Barsnes — APPROVED** [review]

**Pull request merged**